<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dui</class>
 <widget class="QWidget" name="Dui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>405</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>萝莉体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>303</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="SimpleCardWidget" name="SimpleCardWidget">
      <property name="minimumSize">
       <size>
        <width>225</width>
        <height>270</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>270</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="TreeWidget" name="TreeWidget">
         <property name="font">
          <font>
           <family>萝莉体</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QTreeView {
    background-color: transparent;
    border: none;
    border-radius: 5px;
    outline: 0;
    padding-right: 5px;
    font: 12px '萝莉体';
    selection-background-color: transparent;
}

QTreeView[isBorderVisible=true] {
    border: 1px solid rgba(0, 0, 0, 15);
}

QTreeView::item {
    padding: 4px;
    margin-top: 2px;
    margin-bottom: 2px;
    padding-left: 20px;
    border-radius: 5px;
    /* color: black; */
}

QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
    image: url(:/qfluentwidgets/images/tree_view/TreeViewClose_black.svg);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings {
    image: url(:/qfluentwidgets/images/tree_view/TreeViewOpen_black.svg);
}

QTreeView:disabled {
    color: rgba(0, 0, 0, 110);
}

QTreeView::indicator {
    width: 18px;
    height: 18px;
    border-radius: 5px;
    border: none;
    background-color: transparent;
}


QHeaderView {
    background-color: transparent;
}

QHeaderView::section {
    background-color: transparent;
    color: rgb(96, 96, 96);
    padding-left: 5px;
    padding-right: 5px;
    border: 1px solid rgba(0, 0, 0, 15);
    font: 15px '萝莉体';
}

QHeaderView::section:horizontal {
    border-left: none;
    height: 30px;
}

QTreeView[isBorderVisible=true] QHeaderView::section:horizontal {
    border-top: none;
}

QHeaderView::section:horizontal:last {
    border-right: none;
}

QHeaderView::section:vertical {
    border-top: none;
}

QHeaderView::section:checked {
    background-color: transparent;
}

QHeaderView::down-arrow {
    subcontrol-origin: padding;
    subcontrol-position: center right;
    margin-right: 6px;
    image: url(:/qfluentwidgets/images/table_view/Down_black.svg);
}

QHeaderView::up-arrow {
    subcontrol-origin: padding;
    subcontrol-position: center right;
    margin-right: 6px;
    image: url(:/qfluentwidgets/images/table_view/Up_black.svg);
}

QTableCornerButton::section {
    background-color: transparent;
    border: 1px solid rgba(0, 0, 0, 15);
}

QTableCornerButton::section:pressed {
    background-color: rgba(0, 0, 0, 12);
}</string>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="autoExpandDelay">
          <number>-1</number>
         </property>
         <property name="indentation">
          <number>15</number>
         </property>
         <property name="rootIsDecorated">
          <bool>true</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">可选矩阵</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>特殊矩阵</string>
          </property>
          <item>
           <property name="text">
            <string>  3x5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>  5x3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>  5x5</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>二幂矩阵</string>
          </property>
          <item>
           <property name="text">
            <string>  2x2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>  4x4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>  8x8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16x16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32x32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>64x64</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="PushButton" name="PushButton_SF">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>萝莉体</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>加 入 待 处 理 列 表</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
